README.md

```diff
@@ -4,6 +4,16 @@
 
 _This product is in Alpha and not yet ready for production use. We welcome all feedback!_
 
+
+## Try it out!
+
+This script is not perfect but if you're on a linux-based system you should be able to get started by copying the following command into a terminal window and following the prompts:
+`
+curl -s https://raw.githubusercontent.com/DataBiosphere/job-manager/cjl_quick_start/deploy/quickstart/quick_start.py > /tmp/jmui.py && python3 /tmp/jmui.py
```

Comment:
We probably want a stable, non-feature-branch URL if this is accepted and merged.

Reply:
We can modify the URL here to use master branch before this PR gets merged in.